src/OrchardCore.Modules/OrchardCore.OpenId/Views/Items/OpenIdServerDeploymentStep.Thumbnail.cshtml

```diff
@@ -0,0 +1,4 @@
+@model dynamic
+
+<h4 class="card-title">@T["OpenId"]</h4>
+<p>@T["Exports all OpenId server settings."]</p>
```

Comment:
`"OpenId Server"`

Just because there will be more Open Id features that need deployment steps, so it needs to be explicit.